src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,41 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final long ID_OF_DAEWOO = 1;
+    private static final long ID_OF_SCODA = 2;
+    private static final long ID_OF_LEXUS = 3;
+    private static final long ID_OF_TOYOTA = 4;
+    //    private static final Injector injector =
+    //    Injector.getInstance("src/main/java/mate/jdbc");
```

Comment:
```suggestion
        private static final Injector injector = Injector.getInstance("mate.jdbc");
```
did you try this case?